src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -46,24 +46,24 @@
 
 helps['webapp identity assign'] = """
     type: command
-    short-summary: (PREVIEW) assign managed service identity to the webapp
+    short-summary: assign or disable managed service identity to the webapp
     examples:
         - name: assign local identity and assign a reader role to the current resource group.
           text: >
-            az webapp identity assign -g MyResourceGroup -n MyUniqueApp --role reader --scope /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/MyResourceGroup
+            az webapp identity assign -g MyResourceGroup -n MyUniqueApp
         - name: disable the identity when there is need.
           text: >
-            az webapp config appsettings set -g MyResourceGroup -n MyUniqueApp --settings WEBSITE_DISABLE_MSI=true
+            az webapp identity assign -g MyResourceGroup -n MyUniqueApp--disable-msi
```

Comment:
Per executive review a few months ago, that `msi` is discouraged as the term was taken by other groups, hence we switched to use `identity`. So, please rename to `--disable-identity`